src/center.css

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Think again about css class names you used. They are not descriptive at all. A good naming convention is [BEM](getbem.com/naming/).